src/frontend/hooks/persistedState/usePersistedDraftObservation/index.ts

```diff
@@ -184,7 +206,8 @@ export const usePersistedDraftObservation = createPersistedState(
 export const usePreset = () => {
   const {data: presets} = usePresetsQuery();
   const tags = usePersistedDraftObservation(store => store.value?.tags);
-  return !tags ? undefined : matchPreset(tags, presets);
+  const preset = usePersistedDraftObservation(store => store.preset);
+  return preset || !tags ? undefined : matchPreset(tags, presets);
```

Comment:
don't remember operator precedent rules off the top of my head so just checking: is this the intention?

`preset || (!tags ? undefined : matchPreset(...))`

Reply:
Good point, I re-wrote this so that the intention is clearer.